### ISSUE TITLE
Not installing on Apple Watch since upgrade to 14.2

### ISSUE THREAD

**Issue by pixelpants** [nightscout/nightguard#130]
After updating to iOS 14.2 on the phone, the app stopped working on the watch. I restarted both devices, and deleted/reinstalled the app on the phone but the problem persists. The trouble appears when, in the Watch app, when I click 'install'. It appears to install but suddenly stops and stays uninstalled.

**Comment by dhermanns** [comment]
Hi!

I would bet that you have Series 0,1 or 2?

pixelpants <notifications@github.com> schrieb am Mi. 11. Nov. 2020 um 16:57:

**Comment by pixelpants** [comment]
Hi! Alas, my watch is a series 3.

**Comment by dhermanns** [comment]
That should still do. Which is your nightguard Version?

pixelpants <notifications@github.com> schrieb am Mi. 11. Nov. 2020 um 17:07:

**Comment by pixelpants** [comment]
Freshly downloaded from the app store today: 3.1.3.620

**Comment by dhermanns** [comment]
You could try the TestFlight then:
https://testflight.apple.com/join/Ub8WcpcE


pixelpants <notifications@github.com> schrieb am Mi. 11. Nov. 2020 um 17:10:

**Comment by pixelpants** [comment]
Well that worked, in terms of installing on the watch. However, it's not displaying any data. Here is the screenshot of the app status on the watch. Nothing looks wrong.

![nightguard](https://user-images.githubusercontent.com/3606447/98838860-d070d980-2412-11eb-9e41-cf48e6b46e26.png)

**Comment by dhermanns** [comment]
Did you reenter your nightscout URI in the iOS App? Looks like it doesn‘t
make it to your watch. This is still error prone...

pixelpants <notifications@github.com> schrieb am Mi. 11. Nov. 2020 um 17:42:

**Comment by pixelpants** [comment]
Yep, it's working beautifully on the phone. I've left the app running on the watch but there is still no data.

![IMG_3541](https://user-images.githubusercontent.com/3606447/98841407-0f545e80-2416-11eb-9e48-1d88432cdcb5.PNG)

**Comment by dhermanns** [comment]
You should try to reenter the Uri. Or reinstall the watch/ios part.
Sometimes killing the watch app helps too.

pixelpants <notifications@github.com> schrieb am Mi. 11. Nov. 2020 um 18:05:

**Comment by pixelpants** [comment]
Funny thing... I killed the app on the watch and then restarted it. Now I'm getting data.... however, I'm only getting the current blood glucose number - no other data - and, it's not in the mg/dL despite setting it manually in the app.

We're getting closer but not quite there!

**Comment by dhermanns** [comment]
Are you using https?

pixelpants <notifications@github.com> schrieb am Mi. 11. Nov. 2020 um 18:51:

**Comment by pixelpants** [comment]
Yep.

**Comment by pixelpants** [comment]
So, I’ve deleted the TestFlight version of the app and reinstalled the current app store version. As before, it refused to install the app on the watch. So, I went to the App Store via the watch and (after many AppleID hassles), I managed to download the app to my watch. It’s working as beautifully as ever now. 

Can’t quite explain what happened beyond some sort of iOS 14.2 glitch but it’s resolved for now. Thanks for all your help.
